ALBS-40: Migrate PES from SQLite to PostgreSQL

soksanichenko opened the pull request:
- The service migrated from SQLite to PostgreSQL
- And the service migrated from uwsgi to gunicorn

Commit:
ALBS-40: Migrate PES from SQLite to PostgreSQL

- The service migrated from SQLite to PostgreSQL
- And the service migrated from uwsgi to gunicorn

Commit:
ALBS-40: Migrate PES from SQLite to PostgreSQL

- The service migrated from SQLite to PostgreSQL
- And the service migrated from uwsgi to gunicorn

anfimovdm commented on src/backend/db/utils.py:
```diff
@@ -16,35 +16,36 @@
+# def set_sqlite_pragma(dbapi_connection, connection_record):
+#     cursor = dbapi_connection.cursor()
+#     cursor.execute("PRAGMA foreign_keys=ON")
+#     cursor.close()
```
Is that mistake?

soksanichenko replied:
Yep, I removed it.

Commit:
ALBS-40: Migrate PES from SQLite to PostgreSQL

- The service migrated from SQLite to PostgreSQL
- And the service migrated from uwsgi to gunicorn

rpisarev commented on ci/Dockerfile.backend:
```diff
@@ -1,4 +1,5 @@
 FROM python:3 AS python39-uwsgi
+RUN apt-get install -y libpq-dev
```
Maybe we should update the repo before installing and apt-get clean all after?

rpisarev commented on ci/Dockerfile.postgres:
```diff
@@ -0,0 +1,3 @@
+FROM postgres:12.3 as postgres-with-pgloader
+RUN apt-get update
+RUN apt-get install -y wget sudo pgloader libpq-dev
```
And removing a garbage again

Commit:
ALBS-40: Migrate PES from SQLite to PostgreSQL

- The service migrated from SQLite to PostgreSQL
- And the service migrated from uwsgi to gunicorn
